sonarqube/tests/api_responses/metrics_search_p_1

```diff
@@ -0,0 +1,1016 @@
+{
```

Comment:
Can you minify these JSON files?

Reply:
It could be done, but the actual json is based on actual responses from sonarqube.
I think that in any case it could be evaluated and changed if we consider it necessary in a separate PR later.